conduit-extra/Data/Conduit/Binary.hs

```diff
@@ -167,14 +170,31 @@ sourceIOHandle :: MonadResource m
 sourceIOHandle alloc = bracketP alloc IO.hClose sourceHandle
 
 -- | Stream all incoming data to the given 'IO.Handle'. Note that this function
--- will /not/ automatically close the @Handle@ when processing completes.
+-- does /not/ flush and will /not/ close the @Handle@ when processing completes.
 --
 -- Since 0.3.0
 sinkHandle :: MonadIO m
            => IO.Handle
            -> Consumer S.ByteString m ()
 sinkHandle h = awaitForever (liftIO . S.hPut h)
 
+-- | Stream incoming builders, executing them directly on the given 'IO.Handle'
+-- buffer. Note that this function does /not/ automatically close the @Handle@
+-- when processing completes. Pass 'Data.ByteString.Builder.Extra.flush'
+-- to flush buffer.
```

Comment:
Can you provide a minor version bump in the cabal file, update the `ChangeLog`, and add `@since` comments to all new identifiers and instances?

Reply:
Done.

Reply:
Can you also include the changelog update? And it's not necessary, but for future PRs it's better to use the new `@since` syntax for Haddock comments.

Reply:
Right, forgot about changelog

Reply:
I've addressed the points and rebased.